test/cpp/test_aten_xla_tensor_5.cpp

```diff
@@ -261,6 +261,9 @@ TEST_F(AtenXlaTensorTest, TestEmbedding) {
                                            /*scale_grad_by_freq=*/false,
                                            /*sparse=*/false);
     AllClose(b, xla_b);
+    ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
+    ExpectCounterChanged("xla::embedding_symint",
```

Comment:
nit: `xla::embedding_syntint` -> `xla::embedding*`. Not blocker, we can update in the EmbeddingBag PR.

Reply:
Also want to confirm one thing -- before implementing this lowering (i.e. without the changes in this PR), does this metric assertion fail as expected? Just wanted to make sure that the `_symint` variant is getting recognized properly. It should, but just wanted to confirm.

But not a blocker, we can follow-up in the EmbeddingBag PR.

Reply:
Yes, without the change this check fails.

![image](https://github.com/pytorch/xla/assets/15787503/7e4a8797-5237-488e-b159-5ca80463b5c5)